run_dir/design/assign_roles.html

```diff
@@ -0,0 +1,225 @@
+        <th class="sort" data-sort="role">Role</th>
+      </tr>
+    </thead>
+    <tfoot>
+      <tr>
+        <th class="sort" data-sort="user_name">User Name</th>
+        <th class="sort" data-sort="role">Role</th>
+      </tr>
+    </tfoot>
+    <tbody class="list" id="user_table_body">
+    </tbody>
+  </table>
+
+  <div id="createUserModal" class="modal fade">
+    <div class="modal-dialog modal-md">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
+          <h4>Create New User for Genomics-Status </h4>
+        </div>
+        <div class="modal-body">
+          <form id="createUserForm" class="form form-search" role="form">
+            <div class="row">
+              <div class="col-sm-11">
+                <p><b>Create new user.</b></p>
+              </div>
+            </div>
+            <div class="project_page_border_style">
+              <div class="form-group row">
+                <div class="form-group col-sm-6 mx-auto">
+                  <div class="text-center">
+                    <div class="row">
+                      <label for="userName" class="col">Name</label>
+                    </div>
+                    <div class="row">
+                      <div id="userName" class="col-xs-12">
+                        <input class="form-control" id="formCreateUser" placeholder="Name..." type="text">
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div class="form-group col-sm-6 mx-auto">
+                  <div class="text-center">
+                    <label for="roles_dropdown" class="control-label">Choose Role</label>
+                    <div id="roles_dropdown" class="changingDropdown">
+                      <button class="btn btn-default btn-sm dropdown-toggle wrapStyle" type="button" id="roles_dropdownBtn" data-toggle="dropdown">
+                        <i class="glyphicon glyphicon-list-alt" data-userrole='user'></i> User <span class="caret"></span>
+                      </button>
+                      <ul id="roles_dropdownul" class="dropdown-menu dropdown-menu-center dropdown-menu-wide" role="menu" aria-labelledby="roles_dropdownBtn"  style="z-index: 200;">
+                      {% for role, Role in roles.items() %}
+                        <li><a href="#" class="clickDropdownGetValue" data-userrole="{{role}}">{{Role}}</a></li>
+                      {% end %}
+                      </ul>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="form-group row">
+                <div class="form-group col-sm-6 mx-auto">
+                  <div class="text-center">
+                    <p style="margin-top:-10px;">Enter user email(only scilifelab addresses)</p>
```

Comment:
There should be a space between "email" and "(". And the sentence should end with full stop.

Reply:
Done!